routes/personal/personal.spec.js

```diff
@@ -39,6 +39,14 @@ describe('Test /personal responses', () => {
       expect(response.headers.location).toEqual('/success')
       expect(response.statusCode).toBe(302)
     })
+
+    test('it redirects to the checkAnswers when posting Yes and having come from the checkAnswers page', async () => {
+      const response = await request(app)
+        .post('/personal/name').query({ref: 'checkAnswers'})
```

Comment:
Let's either do 
-  `post().query().send()` or 
- `post().send().query()` 

just to be consistent.